cmd/manager/aggregator.go

```diff
@@ -312,9 +321,9 @@ func aggregator(cmd *cobra.Command, args []string) {
 
 	aggregatorConf := parseAggregatorConfig(cmd)
 
-	config, err := rest.InClusterConfig()
+	config, err := config.GetConfig()
```

Comment:
Nit: Can you rename either the variable name or add an import alias? Right now the variable name shadows the import name.

Reply:
Done.